assets/js/app.min.js

```diff
@@ -1 +1 @@
.y=this.y+n,this.y>t.h&&this.reconfig()},"function"!=typeof Object.create&&(Object.create=function(e){function t(){}return t.prototype=e,new t});var getDimensions=function(){var e,t;return"number"==typeof window.innerWidth?(e=window.innerWidth,t=window.innerHeight):document.documentElement&&(document.documentElement.clientWidth||document.documentElement.clientHeight)?(e=document.documentElement.clientWidth,t=document.documentElement.clientHeight):document.body&&(document.body.clientWidth||document.body.clientHeight)&&(e=document.body.clientWidth,t=document.body.clientHeight),{w:e,h:t}},resize=function(){var e=getDimensions();canvas.setAttribute("width",e.w),canvas.setAttribute("height",e.h),Logger.info("[Backdrop] Resized the canvas to fit the viewport.")};window.addEventListener("resize",resize),resize(),window.celestials=[],r(draw),window.meew0Mode={count:0,blocked:!1,askedBlock:!1};var meewButton=document.getElementById("meew-mode-button"),eyes=new Image;if(eyes.src=config.cdnBase+"assets/img/eyes.png",meewButton.onclick=incrementMeew0Mode,Mousetrap.bind("m",incrementMeew0Mode),Mousetrap.bind("l",function(){config.lightTheme?(document.body.className="",config.lightTheme=!1,Logger.info("[meew0 mode] Disabled light theme.")):(document.body.className+="light-theme",config.lightTheme=!0,Logger.info("[meew0 mode] Enabled light theme."))}),config.visualizer&&config.audio){var context=new AudioContext;window.visualizer={toggled:!1,context:context,analyser:context.createAnalyser()},visualizer.analyser.connect(context.destination),visualizer.analyser.fftSize=2048;var source=context.createMediaElementSource(document.getElementById("audio"));source.connect(visualizer.analyser),document.getElementById("visualizer-toggle").onclick=toggleVisualiser,Mousetrap.bind("v",toggleVisualiser),query.hasOwnProperty("visualizer")&&setTimeout(function(){visualizer.toggled=!0,renderVisualizer()},2e3)}config.audio&&r(function(){function e(){var n=config.audioFiles[Math.floor(Math.random()*config.audioFiles.length)];return i===n?e():(Logger.info("[Player] Changing song..."),void t(n))}function t(t){i=t,o.innerText=t[0],o.setAttribute("href",t[1]),n.setAttribute("src","assets/music/"+t[0]+".mp3"),n.load(),n.play(),n.addEventListener("ended",e),Logger.info("[Player] Started playing "+t[0]+".")}var n=document.getElementById("audio"),o=document.getElementById("song"),i=null;return n.volume=config.audioVolumeDefault,document.getElementById("skip-song").onclick=e,Mousetrap.bind("s",e),Mousetrap.bind("up",function(e){e.preventDefault(),n.volume=1==n.volume?n.volume:Math.round(10*(n.volume+.1))/10,Logger.info("[Player] Volume incremented: "+n.volume)}),Mousetrap.bind("down",function(e){e.preventDefault(),n.volume=0==n.volume?n.volume:Math.round(10*(n.volume-.1))/10,Logger.info("[Player] Volume decremented: "+n.volume)}),query.hasOwnProperty("song")&&config.audioFiles[query.song]?(Logger.info("[Player] Playing song from query string."),t(config.audioFiles[query.song])):void e()});
```

Comment:
This is autogenerated when compiling, you don't need to update this manually.

Reply:
Cool, should I remove it from this pr?

Reply:
👍